index.js

```diff
@@ -1,58 +1,108 @@
 // implement your API here
 const express = require("express");
 const server = express();
-const db = require("./data/db.js");
 const port = 8000;
+const db = require("./data/db.js");
+const cors = require("cors");
+
+server.use(express.json());
+server.use(cors());
+
+const sendError = (res, status, msg) => {
+  res.status(status).json({ error: msg });
+  return;
+};
+
+const sendSuccess = (res, status = 200, data, msg) => {
+  res.status(status).json({ data, message: msg });
+  return;
+};
 
 server.get("/", (req, res) => {
   res.send("Hello World, from Node.js!!!!!");
 });
 
 server.post("/api/users", (req, res) => {
-  res.status(201).json({ url: "/api/users", operation: "POST" });
+  const { name, bio } = req.body;
+  if (!name || !bio) {
+    sendError(res, 400, "Provide name and bio");
+    return;
+  }
+  db.insert({ name, bio })
+    .then(response => {
+      sendSuccess(res, 201, response, "User successfully added!");
+    })
+    .catch(() => {
+      sendError(
+        res,
+        500,
+        "There was an error while saving the user to the database"
+      );
+      return;
+    });
 });
 
 server.get("/api/users", (req, res) => {
   db.find()
     .then(users => {
-      res.json(users);
+      // res.json(users);
+      sendSuccess(res, 200, users, "Successfully retrieved list of users!");
```

Comment:
This method is not flexible enough as it would need to be modified if you wanted to send something else other than what is included. Such as an object of data or more messages. Try not to lose flexibility and functionality when trying to stay DRY.